tools/slo-generator/slo_generator/exporters/stackdriver.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+`stackdriver.py`
+Stackdriver Monitoring exporter class.
+"""
+import logging
+from google.cloud import monitoring_v3
+from slo_generator.exporters.base import Exporter
+
+LOGGER = logging.getLogger(__name__)
+DEFAULT_METRIC_TYPE = "custom.googleapis.com/error_budget_burn_rate"
+DEFAULT_METRIC_DESCRIPTION = ("Speed at which the error budget for a given"
+                              "aggregation window is consumed")
+
+class StackdriverExporter(Exporter):
+    """Stackdriver Monitoring exporter class."""
+    def __init__(self):
+        self.client = monitoring_v3.MetricServiceClient()
+
+    def export(self, data, **config):
+        """Export data to Stackdriver Monitoring.
+
+        Args:
+            data (dict): Data to send to Stackdriver Monitoring.
+            config (dict): Stackdriver Monitoring metric config.
+                project_id (str): Stackdriver host project id.
+                custom_metric_type (str): Custom metric type.
+                custom_metric_unit (str): Custom metric unit.
+
+        Returns:
+            object: Stackdriver Monitoring API result.
+        """
+        self.create_metric_descriptor(data, **config)
+        self.create_timeseries(data, **config)
+
+    def create_timeseries(self, data, **config):
+        """Create Stackdriver Monitoring timeseries.
+
+        Args:
+            data (dict): Data to send to Stackdriver Monitoring.
+            config (dict): Metric config.
+
+        Returns:
+            object: Metric descriptor.
+        """
+        series = monitoring_v3.types.TimeSeries()
+        series.metric.type = config.get('metric_type', DEFAULT_METRIC_TYPE)
+
+        # Write metric labels
```

Comment:
nit: Here and everywhere, comments should be full sentences (which means ending w/ period)

Reply:
Done.